bin/hyperlog.js

```diff
@@ -142,17 +133,17 @@ const writeLine = async (path, line) => {
   return fs.promises.writeFile(path, data, options)
 }
 
-const prepareOutputDestination = async (path, multifile = false) => {
-  const dirCreated = multifile ? await createDir(path) : await createFileDir(path)
+const prepareOutputDestination = async (path) => {
+  const isDirectory = await isDir(path)
```

Comment:
suggestion re improvement
let's say user has this case:
```
node bin/hyperlog.js read --key 328625336c9063f5c725401c0c428cbfb559bef32852a773bd4d828b81565c04 --output tmp-logs/
```
currently in above case we're writing logs to file `tmp-logs`, maybe we can add some checks if it has parts like `/`, `\` or `..` to set `isDirectory` to `true`?

something like this
```js
let isDirectory = await isDir(path)
if (!isDirectory) isDirectory = ['\\', '/', '..'].some(x => path.includes(x))

Reply:
> maybe we can add some checks if it has parts 

shouldn't we check only for the trailing slash? Since the file path can also contain the above characters